prow/plugins/trigger/pr.go

```diff
@@ -72,14 +72,16 @@ func handlePR(c client, pr github.PullRequestEvent) error {
 }
 
 func askToJoin(ghc githubClient, pr github.PullRequest) error {
-	commentTemplate := `
-Can a [%s](https://github.com/orgs/%s/people) member verify that this patch is reasonable to test? If so, please reply with "@k8s-bot ok to test" on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands will still work. Regular contributors should join the org to skip this step.
+	commentTemplate := `Hi %s. Thanks for your PR.
+
+I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that this patch is reasonable to test. If it is, they should reply with "@k8s-bot ok to test" on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should join the org to skip this step.
```

Comment:
Maybe change this to backtick k8s-bot ok to test so that it looks more like a command?

Also consider directing them to slack if they have questions.